app/src/main/java/de/westnordost/streetcomplete/data/StreetCompleteOpenHelper.java

```diff
@@ -115,7 +115,8 @@
 				OsmNoteQuestTable.Columns.QUEST_STATUS +	" varchar(255)	NOT NULL, " +
 				OsmNoteQuestTable.Columns.COMMENT +			" text, " +
 				OsmNoteQuestTable.Columns.LAST_UPDATE + 	" int			NOT NULL, " +
-				OsmNoteQuestTable.Columns.NOTE_ID +			" INTEGER		UNIQUE NOT NULL " +
+				OsmNoteQuestTable.Columns.NOTE_ID +			" INTEGER		UNIQUE NOT NULL, " +
+				OsmNoteQuestTable.Columns.IMAGE_PATHS +		" text " +
```

Comment:
**Serialization issue:** If you serialize/deserialize the image paths from a byte[] blob (using `serializer`), this needs to be of type `blob`, not `text`.

Reply:
Done